mappings/net/minecraft/data/family/BlockFamilies.mapping

```diff
@@ -50,6 +50,8 @@ CLASS net/minecraft/class_5793 net/minecraft/data/family/BlockFamilies
 	FIELD field_28948 DEEPSLATE_TILE Lnet/minecraft/class_5794;
 	FIELD field_28949 DEEPSLATE_BRICK Lnet/minecraft/class_5794;
 	FIELD field_29079 COBBLED_DEEPSLATE Lnet/minecraft/class_5794;
+	FIELD field_33117 WOODEN_KEY Ljava/lang/String;
+	FIELD field_33118 HAS_PLANKS_KEY Ljava/lang/String;
```

Comment:
Names based on the contents of their assigned values. `wooden` and `has_planks` respectively. Key because it's a string, not a `BlockFamily`